README.md

```diff
@@ -1,13 +1,74 @@
-# Taskmaster [![42](https://i.imgur.com/9NXfcit.jpg)](i.imgur.com/9NXfcit.jpg)
 
-Taskmaster is a 42 project. It is an implementation of a **process supervisor** and aims at getting familiar with job control mechanisms on UNIX-like operating systems.
+<br />
+<p align="center">
+  <a href="https://github.com/pscott/TaskMaster">
+    <img src="assets/logo.png" alt="Logo" width="90" height="90">
+  </a>
 
----
+  <p align="center">
+    <img src="assets/title.png" alt="TaskMaster" width="280" height="70">
+  </p>
+
+  <p align="center">
+    A process control system for UNIX.
+    <br />
+    <a href="https://github.com/pscott/TaskMaster">View Demo</a>
+    ·
+    <a href="https://github.com/pscott/TaskMaster/issues">Report Bug</a>
+    ·
+    <a href="mailto:abarthel@student.42.fr">Request Feature</a>
```

Comment:
Usually features are requested by creating a new issue :p
I think we'd be better off redirecting to /issues

Reply:
True